src/extension.ts

```diff
@@ -160,7 +179,18 @@ class GithubTimeline implements TimelineProvider, Disposable {
 			window.showInformationMessage('Could not authenticate your GitHub!');
 			return;
 		}
-		const response: any = await queryService.getPullRequest(session);
+
+		let owner = 'microsoft';
+		let repo = 'vscode';
+		let number = '116984';
```

Comment:
Curious... Why did you decide on the default values rather than throwing some error if invalid?

Reply:
I just didn't want to change the existing behavior